### DISCUSSION TITLE
Dependabot go.mod is not tidy

### DISCUSSION THREAD

**Discussion by kaovilai** [community/community#54411] [Code Security] [answered]
### Select Topic Area

Bug

### Body

https://github.com/kaovilai/udistribution/actions/runs/4879007014/jobs/8705170542?pr=28#step:10:7

Per https://github.blog/changelog/2020-10-19-dependabot-go-mod-tidy-and-vendor-support/
This should not be the case.

**Comment by carogalvin** [comment, marked as answer]
Hi @kaovilai , thanks for sharing this feedback! Sometimes, Dependabot is not successful when running `go mod tidy`, but we do not fail the Dependabot update job when this happens, which is why you're seeing this. If you would like to investigate a fix, I would invite you to either submit a support ticket or even explore fixing the code yourself in our [open source Dependabot repo](https://github.com/dependabot/dependabot-core). Thanks!
